README.md

```diff
@@ -48,11 +48,16 @@ Installation process for the setting up Wordpress on a local environment.
       git clone git@github.com:chapmanu/chap-press.git
       cd chap-press/public
       cp -v wp-config.php{-dist,}
-      wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
 
       # The wp-config file already contains the database information to get started.
       # These are for local development only.
 
+- **Connect Wordpress to DB**
+	
+	While inside `/public` folder
+	  brew install wp-cli
+	  wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
+
```

Comment:
Not sure if this works or not. This is what I got after installing `wp-cli`:

```
$ wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
Error: This does not seem to be a WordPress install.
Pass --path=`path/to/wordpress` or run `wp core download`.
SMC-MBOOKP13-01:chap-press atwell$ cd public/
SMC-MBOOKP13-01:public atwell$ wp core install --url=http://localhost:8222/ --title=chap-press --admin_user=chappress --admin_password=password --admin_email=chappress@gmail.com
WordPress is already installed.
```

I'd prefer everything was run from project root without having to change directories.